### ISSUE TITLE
Suggestion to build an ETC halving countdown website similar to BTC/LTC

### ISSUE THREAD

**Issue by TheEnthusiasticAs** [ethereumclassic/ECIPs#255]
People often ask me in telegram group, when the halving is. Such a countdown would handy:
https://www.bitcoinblockhalf.com/.
I am not a computer engeneer. A contribution by someone with knowledge would be appreciated. Thank You!

**Comment by hitchcott** [comment]
I'd love to do this :D

**Comment by gitr0n1n** [comment]
Suggestions: host this on the ethereumclassic.org/countdown url. So we can host a centralized place for many countdowns:
- 5m20 reduction and future emission schedule
- Aztlan upgrade
- Accepted ECIPs

Then we can set up some redirects for domains specific to each countdown to hype that event. Example:
- ECIP1017.org
- 5M20.org

- ETCcountdown.org
- ETCclock.com
- ETCevent.com

- EtherEmission.com
- EmissionSchedule.com

**Comment by hitchcott** [comment]
Yes. A place with many countdowns is a good idea.

**Comment by TheEnthusiasticAs** [comment]
I like the idea of @gitr0n1n

**Comment by TheEnthusiasticAs** [comment]
Putting the released domains here:
5m20 = https://etcis.money
ECIP1017 = https://ecip1017.site

**Comment by gitr0n1n** [comment]
I love this since we will always have a countdown for users. Once we hit era3, we countdown to era4....then era5.....

Long term use in this. Would be great to have it on the ethereumclassic.org site.

5m20.ethereumclassic.org ?
era.ethereumclassic.org ?
countdown.ethereumclassic.org ?

Not sure whats the best fit, but feels like a big win for the site.

**Comment by TheEnthusiasticAs** [comment]
I would take the first one. It is unique

**Comment by gitr0n1n** [comment]
@hitchcott When do you estimate you'll have the spare time to release an alpha version of the countdown?

**Comment by gitr0n1n** [comment]
This could be moved to the web development repo to remove clutter from the buzzing ECIP repo.

@TheEnthusiasticAs @hitchcott

**Comment by realcodywburns** [comment]
is this thread still active? 
Moving to close on 10/1/2020 if no objections

**Comment by TheEnthusiasticAs** [comment]
Yes, it is active @realcodywburns. It is waiting for someone to champion it.

**Comment by gitr0n1n** [comment]
I think @hitchcott is busy in his life right now, but it sounded like he wanted to push this when he has the bandwidth.

**Comment by TheEnthusiasticAs** [comment]
yes, you are right, it should be moved to website repo.
